DEVELOPER_GUIDE.md

```diff
@@ -50,7 +50,7 @@ sdk install java 14.0.2-open
 sdk use java 14.0.2-open
```

Comment:
should these be updated as well?

Reply:
yep

Reply:
I'm not familiar with this `sdk` tool, so I wasn't sure what to do here. By only requiring the LTS java 11, the user could probably just avoid this tool and use their normal system package manager?

Reply:
It's here as just an example, swap 14 -> 11 and we're good to go?

Reply:
I'd prefer to remove it. I have no idea how to test my instructions. Sorry if it comes across as rude, but I'm seriously not gonna run any `curl | bash` stuff.

Reply:
deleting the block makes more sense to me
there are other ways to manage different versions of java locally

Reply:
FWIW I added this because I was trying to make it super simple for someone to start contributing and the sdk tool is super popular in my world, along with the equivalents of `nvm` for node or `rvm` for ruby. I am always puzzled why people hand-install SDKs when this is so much easier. Obviously `| sh` anything can be replaced by a download or a `brew install` for Mac or whatever.

No strong feelings if you want to just remove this as part of this PR.

Reply:
ok, ill nuke it. I think telling the user how to install java is helpful when you're asking someone to install a specific EOL/unsupported version of java such as 14: it is indeed obscure to locate and install it, it's been unsupported since september 2020.

But, now that problem goes away if the build works with LTS 11+: really things will just work. I don't install any software outside of package manager for a large number of reasons (it will be maintained and patched, stay compatible across upgrades, get security fixes, get catalogued properly in the local package registry, dependencies will be taken care of, ...). 

I really have no idea what this sdkman.io is going to do to my computer (even looking at the script it runs, which seems to download more code and then run that).

java 11 is widely available via any standard package manager, whether that's ancient centos 7 yum or bleeding edge arch's pacman. And if you want to install java manually, its easy to get from adoptopenjdk too. So java 11 is a way simpler ask on any user and I don't think we need to instruct users on how to get it.

Reply:
just as a side, as far as this specific tool piping curl through bash, I am also super suspicious about stuff like that, and don't think we should suggest it to users. I know there are some defenders of the `curl | bash`, yet we still problems from this stuff such as https://thenewstack.io/not-your-usual-supply-chain-hack-the-codecov-bash-uploader-blunder/

at my job at least, given recent supply chain attacks in the news, there is a lot of tightening of controls around this kind of stuff.

Reply:
Thanks for calling this out.